rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -27,10 +27,10 @@
 #include <regex>
 
 #include "rcutils/filesystem.h"
+#include "rcutils/format_string.h"
```

Comment:
You'll need an `#include <dds/features.h>` somewhere in here. Or I guess `#include "dds/features.h"` to keep the style.

Reply:
Added, but this file does not exist yet--waiting for https://github.com/eclipse-cyclonedds/cyclonedds/pull/448

Reply:
It has now been merged.